adafruit_bh1750.py

```diff
@@ -97,15 +104,15 @@ class RWBitfields:
 
     """
 
-    def __init__(self, num_bits, lowest_bit):
+    def __init__(self, num_bits: int, lowest_bit: int) -> None:
         self._bit_mask = ((1 << num_bits) - 1) << lowest_bit
         self._lowest_bit = lowest_bit
 
-    def __get__(self, obj, objtype=None):
+    def __get__(self, obj: object, objtype: Optional[type] = None) -> int:
 
         return (obj._settings & self._bit_mask) >> self._lowest_bit
 
-    def __set__(self, obj, value):
+    def __set__(self, obj: object, value: int) -> None:
```

Comment:
`__set__()` only activates when used from an instance, so no worries about a `None` value.  Again, we can be specific since it should only be used within `BH1750`, so the annotation is:

```suggestion
    def __set(self, obj: "BH1750", value: int) -> None:
```